gluten-core/src/main/scala/io/glutenproject/extension/OthersExtensionOverrides.scala

```diff
@@ -35,5 +35,8 @@ object OthersExtensionOverrides extends GlutenSparkExtensionsInjector {
     BackendsApiManager.getSparkPlanExecApiInstance
       .genExtendedStrategies()
       .foreach(extensions.injectPlannerStrategy)
+    BackendsApiManager.getSparkPlanExecApiInstance
+      .genInjectedFunctions()
+      .foreach(extensions.injectFunction)
```

Comment:
Same issue here: is the extension able to be called from Spark executor? In the case of running a Spark cluster.

Reply:
If not, probably we can try if C++ side VeloxInitializer is OK to do the initialization.

Reply:
You are right. I assume it cannot.